web/e2e-tests/stream_create.test.ts

```diff
@@ -81,6 +81,14 @@ async function create_stream(page: Page): Promise<void> {
         stream_description: "Everything Puppeteer",
     });
     await page.click("form#stream_creation_form .finalize_create_stream");
+    // an explanatory modal is shown for the first stream created
+    await page.waitForSelector("modal__btn dialog_submit_button");
+    await page.click("modal__btn dialog_submit_button");
```

Comment:
Would be better to use `wait_for_micromodal_to_open` here like we do in other tests and use similar selectors.

Reply:
You can look at tests in `user-deactivation.test.ts`.

Reply:
Thanks, I looked at it and used the functions in `common` module for the micromodal.